keras/backend/tensorflow_backend.py

```diff
@@ -1486,6 +1486,15 @@ def l2_normalize(x, axis):
     return tf.nn.l2_normalize(x, dim=axis)
 
 
+def in_top_k(predictions, targets, k):
+    '''Given tensors representing predictions and targets, evaluates whether the
+    targets are within the top k predictions
+    '''
```

Comment:
The docstring should specify that the inputs are supposed to be batches. It should also describe the output (shape, type and contents).